Ignore .DS_Store when clearing storage

For some reason  in the code below

https://github.com/thelounge/thelounge/blob/32f018143357e29a23f057054f70a522fe175feb/server/plugins/storage.ts#L92-L104

`.DS_Store` is recognised as a folder which leads to errors such as

``` 
[ERROR] Could not clear storage folder, ENOTDIR: not a directory, scandir '/var/opt/thelounge/storage/.DS_Store
```

During startup. Can thelounge ignore `.DS_Store` files?

- _Node version:_ v18.20.8
- _Browser version:_ not relevant (backend bug)
- _Device, operating system:_ Macbook Pro, macOS Tahoe 26.0.1, [Podman](https://podman.io/), [thelounge-docker](https://github.com/thelounge/thelounge-docker)
- _The Lounge version:_ v4.4.3

--- a/server/plugins/storage.ts
+++ b/server/plugins/storage.ts
@@ -25,8 +25,9 @@ class Storage {
 			return;
 		}
 
-		// TODO: Use `fs.rmdirSync(dir, {recursive: true});` when it's stable (node 13+)
-		items.forEach((item) => deleteFolder(path.join(dir, item)));
+		items.forEach((item) => {
+			fs.rmSync(path.join(dir, item), {recursive: true, force: true});
+		});
 	}
 
 	dereference(url) {
@@ -88,17 +89,3 @@ class Storage {
 }
 
 export default new Storage();
-
-function deleteFolder(dir: string) {
-	fs.readdirSync(dir).forEach((item) => {
-		item = path.join(dir, item);
-
-		if (fs.lstatSync(item).isDirectory()) {
-			deleteFolder(item);
-		} else {
-			fs.unlinkSync(item);
-		}
-	});
-
-	fs.rmdirSync(dir);
-}

